homeassistant/components/slide_local/config_flow.py

```diff
@@ -0,0 +1,129 @@
+"""Config flow for slide_local integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from goslideapi.goslideapi import (
+    AuthenticationFailed,
+    ClientConnectionError,
+    ClientTimeoutError,
+    DigestAuthCalcError,
+    GoSlideLocal as SlideLocalApi,
+)
+import voluptuous as vol
+
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_API_VERSION, CONF_HOST, CONF_MAC, CONF_PASSWORD
+from homeassistant.helpers.device_registry import format_mac
+from homeassistant.helpers.selector import (
+    BooleanSelector,
+    SelectOptionDict,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+
+from . import SlideConfigEntry
+from .const import CONF_INVERT_POSITION, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+BOOLEAN_SELECTOR = BooleanSelector()
```

Comment:
Why do we have this here?

Reply:
Took this from an example (I guess it was to have all selectors defined at the top). Would it be better to directly use BooleanSelector() below?

Reply:
There's no real benefit, so I would stick with consistency and define it down below